tag_manager/data_layer_name.rst

```diff
@@ -0,0 +1,56 @@
+.. highlight:: js
+.. default-domain:: js
+
+Custom data layer name
+======================
+
+Introduction
+------------
+The purpose of this article is to describe the steps that need to be performed to set up custom data layer name. The default value for the data layer name is `dataLayer`. Since PPMS version 10.0.0, there is a possibility to change that value.
```

Comment:
In public docs we should operate only on MAJOR.MINOR versions since bufixes should not have any effect on API. Here it should be `10.0`.

Additionally you can try using [versionadded](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded) or [versionchanged](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged) message to make it stand out more.

Reply:
I am trying to figure out where to put the `versionadded` directive for it to make sense. Two options that seem to work is the very top of the file or the start of the introduction section. Do you have any other ideas?

Reply:
I would try to add it as a extension to this description. See how it looks (exact text may need consideration):
```
...
The default value for the data layer name is `dataLayer`.

.. versionadded:: 10.0
    It is now possible to change that value.
```